### ISSUE TITLE
Preferences for editor not being saved across sessions

### ISSUE THREAD

**Issue by TS6ix2GaH3q679h5** [WordPress/gutenberg#34405]
I prefer to use the default editor, so untick the "**Use theme styles**" options each time when composing a post. I also untick the **pre-publish checklist**.

However this settings change only is maintained for the session. Quitting the browser and restarting results in all toggles being reset to their original defaults.

<img width="731" alt="image" src="https://user-images.githubusercontent.com/74590442/131459694-b2b52a6e-b57c-40df-b741-4958c56941b5.png">

I was suspicious that it might be a browser cookie blocker or adblocker preventing this, so I disabled all extensions and content blockers. Still getting the same result, both across browsers (Brave, Chrome, Safari) and the standalone (Electron) Wordpress App. Even across my two computers. 

Am I doing something wrong?

**Comment by talldan** [comment]
@TS6ix2GaH3q679h5 The data is stored in browser local storage currently, so will only be persisted in that particular browser and computer. Any settings won't be shared across browsers or computers, though there's some work in progress to improve this and save the settings to the database.

I just gave it a test and didn't have any problem with the preferences being restored after quitting and reopening a browser.

It's curious that you're experiencing this across multiple browsers/devices. I would suggest that it might be worth checking your browser security settings to see if local storage data is being retained. I know in Brave there are settings for this:
<img width="671" alt="Screenshot 2021-08-31 at 4 49 41 pm" src="https://user-images.githubusercontent.com/677833/131472671-857a328c-c894-45df-bbdf-b1b5101c7cf1.png">

But it seems unlikely you'd have the same issue with every browser.

**Comment by TS6ix2GaH3q679h5** [comment]
@talldan OK, I was wondering if these settings were stored in the account settings or only in a cookie. Knowing they are stored locally clearly means something must be up with my home network settings. A cookie must be being deleted somewhere though I can't seem to figure it out. I've tried explicitly whitelisting wordpress.com, mysite.wordpress.com, and public-api.wordpress.com since (as far as I can tell) those are the 3 being set by the editor. Somehow my home network filtering must be responsible. II will keep investigating come back and mark this closed when I figure it out.

Incidentally, as a workaround, I wonder if there is any URL parameter I could bookmark for having the editor set to not use the blog theme styles. Something like ?theme=0 or something? 

Edit: 
I wonder which cookie this setting is saved in. I've tried disabling any network traffic blockers I can think of. I've also tried extracting cookies set by the editor, then comparing them with those set after the switch toggle, and as far as I can tell they are all identical except for the access token.

Meanwhile, I've found that the setting is now being maintained in the Electron version of the Wordpress for Mac app. Maybe I disabled whatever was causing this bug. But still no luck in Safari and Chrome. I wish I could find precise cookie and cookie value that stores this preference.
